canonical/ubuntu.com
# Refactor summarized status logic

## Done

- Refactored summarized statuses logic

## QA

- View the site locally in your web browser at: https://ubuntu-com-14085.demos.haus/security/cves
- See that the summarized statuses are displaying as expected
- Click on any of the "By Ubuntu release" links 
  - See that summarized statuses are correct here as well and that they are hidden when the detailed view switch is toggled

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8197
